ci-operator/step-registry/cucushift/hypershift-extended/capi/enable-hc/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - LiangquanLi930
+  - heliubj18
+reviewers:
+  - LiangquanLi930
+  - heliubj18
```

Comment:
Could you add me as reviewer & approver as well ?

Reply:
ok